pkg/reader/ccip.go

```diff
@@ -136,7 +136,10 @@ func (r *ccipChainReader) CommitReportsGTETimestamp(
 			},
 		},
 		query.LimitAndSort{
-			SortBy: []query.SortBy{query.NewSortByTimestamp(query.Asc)},
+			SortBy: []query.SortBy{query.NewSortBySequence(query.Asc)},
+			Limit: query.Limit{
+				Count: uint64(limit * 2),
```

Comment:
Hmm, why limit*2? :D

Reply:
The limit is applied at the end of the method

```
	if len(reports) < limit {
		return reports, nil
	}
	return reports[:limit], nil
```
I thought maybe have more so that we are sure the count will be higher than the limit.